testdata/project-v3-addon/PROJECT

```diff
@@ -1,5 +1,5 @@
 domain: testproject.org
-layout: go.kubebuilder.io/v3
+layout: go.kubebuilder.io/v3,declarative.kubebuilder.io/v1
```

Comment:
The layout of the project is `go.kubebuilder.io/v3`. The `declarative.kubebuilder.io/v1` is an additional plugin used on top of that and should be tracked in the plugins attr instead: 

```
plugins:
  declarative.kubebuilder.io/v1: {}
```

Reply:
No, `declarative/v1` is configured for the whole project because there is no way to configure a plkugin for only one command call. That is a future feature tracked in #1941.